src/main/java/org/testng/TestRunner.java

```diff
@@ -862,6 +866,16 @@ private void privateRun(XmlTest xmlTest) {
     return result;
   }
 
+  private Map<String, String> getParameters(IMethodInstance im) {
```

Comment:
Fixed this. Made the method static and resorted to using `findMethodParameters()` (That was a good one.. I wasn't aware of that. Thanks for pointing it out)